problem_builder/public/js/mentoring_with_steps.js

```diff
@@ -111,7 +93,8 @@ function MentoringWithStepsBlock(runtime, element) {
                 // We are now showing the review step / end
                 // Update the number of attempts.
                 attemptsDOM.data('num_attempts', response.num_attempts);
-                updateGrade(response.grade_data);
+                reviewStepDOM.html($(response.review_html).html());
```

Comment:
Again a question for my personal education: How is this different from `reviewStepDOM.html(response.review_html)`?  Does it just remove the outermost HTML tag?  If so, why is it included in `review_html` in the first place?